benchmark/utils/cuda_linops.cu

```diff
@@ -349,6 +349,9 @@ private:
 #endif  // CUDA_VERSION < 11000
 
 
+#if CUDA_VERSION < 11030
```

Comment:
As far as I can see, the first deprecation here was 11.2.1: https://docs.nvidia.com/cuda/archive/11.2.1/cusparse/index.html#csrmvEx

Reply:
Thanks! I only checked major versions. Let me update that